src/extra/AutoUI/index.tsx

```diff
@@ -142,6 +148,16 @@ export const AutoUI = <T extends AutoUIBaseResource<T>>({
 
 	React.useEffect(() => {
 		setSelected([]);
+		if (filters.length) {
+			const { pathname } = window.location;
+			history?.replace?.({
+				pathname,
+				search: listFilterQuery(model.schema, filters),
+			});
+		}
+		if (onChange) {
+			onChange({ resource: model.resource, filters });
```

Comment:
Aren't we already doing this in the `Filter`'s `changeFilters` callback (potentially resulting double calls)?
Would defining a single `onChangeFilters = useCallback(...)` function and using it in both `Filter` components allow us to drop this?

Reply:
Seems doing `changeFilters` here is unnecessary so I've removed it